src/Storefront/Page/Product/Review/ProductReviewLoader.php

```diff
@@ -48,126 +40,35 @@ public function __construct(
      */
     public function load(Request $request, SalesChannelContext $context): ReviewLoaderResult
     {
+        Feature::triggerDeprecationOrThrow('v6.7.0.0', Feature::deprecatedClassMessage(self::class, 'v6.7.0.0', 'Use \Shopware\Core\Content\Product\SalesChannel\Review\ProductReviewLoader instead'));
```

Comment:
if the third parameter is given, it will already be part of a sentence like "use ... instead" See https://github.com/shopware/shopware/blob/trunk/src/Core/Framework/Feature.php#L256

```suggestion
        Feature::triggerDeprecationOrThrow('v6.7.0.0', Feature::deprecatedClassMessage(self::class, 'v6.7.0.0', ProductReviewLoader::class));
```

Reply:
Changed.